tubular/tests/test_get_learners_to_retire.py

```diff
@@ -99,7 +99,7 @@ def test_misconfigured(*args, **kwargs):
     mock_get_learners_to_retire = kwargs['learners_to_retire']
 
     mock_get_access_token.return_value = ('THIS_IS_A_JWT', None)
-    mock_get_learners_to_retire.side_effect = HttpClientError
+    mock_get_learners_to_retire.side_effect = RequestException
```

Comment:
Please clarify why we do not specify the exception in the side_effect but use a generic one?